integration_tests/helpers/transactionBuilder.js

```diff
@@ -125,9 +125,10 @@ class TransactionBuilder {
     let scriptOffsetPublicKey = tari_crypto.pubkey_from_secret(
       scriptOffsetPrivateKey.toString("hex")
     );
+    let nop_script_bytes = Buffer.from([0x73]);
 
     let beta = calculateBeta(
-      nopScriptHash,
+      nop_script_bytes,
```

Comment:
I think we prefer `camelCase` for JS, and if lint doesn't complain then it should! Maybe I'm just bikeshedding, but everything else is camelCase I think.

Reply:
fair enough

Reply:
personally I prefer snake_case but 🤷‍♂️